src/AzureClient/Magic/ConnectMagic.cs

```diff
@@ -39,73 +39,91 @@ public ConnectMagic(IAzureClient azureClient)
                 new Documentation
                 {
                     Summary = "Connects to an Azure Quantum workspace or displays current connection status.",
-                    Description = @"
-                            This magic command allows for connecting to an Azure Quantum workspace
-                            as specified by the resource ID of the workspace or by a combination of
-                            subscription ID, resource group name, and workspace name.
+                    Description = $@"
+                        This magic command allows for connecting to an Azure Quantum workspace
+                        as specified by the resource ID of the workspace or by a combination of
+                        subscription ID, resource group name, and workspace name.
 
-                            If the connection is successful, a list of the available Q# execution targets
-                            in the Azure Quantum workspace will be displayed.
-                        ".Dedent(),
+                        If the connection is successful, a list of the available Q# execution targets
+                        in the Azure Quantum workspace will be displayed.
+
+                        #### Required parameters
+
+                        The Azure Quantum workspace can be identified by resource ID:
+                        - `{ParameterNameResourceId}=<string>`: The resource ID of the Azure Quantum workspace.
+                        This can be obtained from the workspace page in the Azure portal.
+
+                        Alternatively, it can be identified by subscription ID, resource group name, and workspace name:
```

Comment:
We should check the reference rendering, but you may need a blank line here for the list to format as a list.
```suggestion
                        Alternatively, it can be identified by subscription ID, resource group name, and workspace name:

```

Reply:
It looks fine in the reference rendering, but yes I'll add these blank lines everywhere, thanks for noticing this.